packages/components/psammead-media-player/src/index.jsx

```diff
@@ -91,6 +91,7 @@ CanonicalMediaPlayer.propTypes = {
     duration: string,
     durationSpoken: string,
     type: oneOf(['video', 'audio']),
+    guidance: string,
```

Comment:
Would `guidanceMsg` or `guidanceMessage` be more appropriate here? We had a prop like this called `placeholder` not long ago, that was causing problems, because it was confused with a `boolean`.